src/views/inspector/InspectorViewProvider.ts

```diff
@@ -349,8 +381,8 @@ export class InspectorViewProvider implements vscode.WebviewViewProvider {
     const projectId = StateManager.getFolderState(folder.name, KEYS.PROJECT_ID)
     const orgId = getOrganizationId(folder)
     const dashboardPath = this.selectedType === 'Variable' ?
-      `variables/${this.variables[this.selectedKey].key}` :
-      `features/${this.features[this.selectedKey].key}`
+      `variables/${this.variables[this.selectedKey]?.key}` :
+      `features/${this.features[this.selectedKey]?.key}`
```

Comment:
will this produce a broken link with undefined/null if the key doesn't exist in the map? If that's the case it may be preferable to either hide the link to at least link to the variable/feature list page instead

Reply:
it should exist at all times, but I can add a default `''` so that it'll just link to the corresponding features or variables page